contributing_to_docs/doc_guidelines.adoc

```diff
@@ -122,6 +122,31 @@ To use a discrete heading, just add `[discrete]` to the line before your unique
 == Managing authorization policies
 ----
 
+=== Creating unique IDs for 'Additional resources' and 'Next steps' titles
+
+Use unique IDs for *Additional resources* and *Next steps* titles. You can append `-additional-resources` or `-next-steps` to a unique string that describes the assembly topic.
+
+The following examples include IDs that are unique to the *Understanding the monitoring stack* assembly.
+
+*Example unique ID for an 'Additional resources' title*
+
+----
+[id="understanding-the-monitoring-stack-additional-resources"]
```

Comment:
Apologies if this has been brought up elsewhere: Is there a reason why we can't use the assembly context variable here instead?
[id="{context}-additional-resources"]
and
[id="{context}-next-steps"] ?
To my mind it simplifies things if the title/filename/context ever changes, and folks can use a generic snippet/copypasta for any assembly. Also less typing :)

Reply:
That sounds like a good idea, from my perspective.

Reply:
It is probably worth highlighting that this would add some additional complexity when determining what a cross-reference to the title would be.

Reply:
To keep this similar to how we would normally have anchor ids defined, this (and the other examples) should be changed to (for example):

`additional-resources_understanding-the-monitoring-stack
`
Keeping with:

`<title>_<context> 
`
syntax.

So the title separated by an underscore. I understand the issue with confusion when creating cross-references and yes happy to provide actual values but we should keep the syntax.

Reply:
@vikram-redhat thank you for highlighting this. I have updated the introduction to the subsection and the examples so that the context is at the end.

I have also added the following line to the end of the first paragraph, to indicate that the context value declared in the assembly can be used:

"The unique string can match the value assigned to the :context: attribute in the assembly."

This is just short of stating that `{context}` could be added to the end of the anchor ID.